translations/translations.json

```diff
@@ -4202,6 +4201,8 @@
       "to": "indtil",
       "resetFilter": "Slet filter:",
       "shouldBeEarlier": "Startdatoen skal ligge før slutdatoen.",
+      "selectStartDateCalendar": "Select start date in the calendar",
+      "selectEndDateCalendar": "Select end date in the calendar",
```

Comment:
:wrench:  These should be in "en" section not "da"

Reply:
Thank you Bahaa for your thorough reviews, seriously! Very good catch again, no idea where my head is at the moment 🤦